src/app.ts

```diff
@@ -0,0 +1,23 @@
+import express from "express";
+import routes from "./routes";
+
+class App {
+  public server;
+
+  constructor() {
+    this.server = express();
+
+    this.middlewares();
+    this.routes();
+  }
+
+  middlewares() {
+    this.server.use(express.json());
+  }
+
+  routes() {
+    this.server.use(routes);
+  }
+}
```

Comment:
Mostly a style comment, but I don't see the value of class here. It feels like you can make this file more succinct with:

```ts
function middlewares(server) {
    server.use(express.json());
}

function routes(server) {
    server.use(routes);
}

const server = express();
middlewares(server);
routes(server);

export default server;
```